### ISSUE TITLE
Pack Fails to Launch

### ISSUE THREAD

**Issue by preflora** [Cuppycake/Mech-and-Magic#124]
(apologies if this is the wrong place for this) I was playing a previous 1.7.10 version of the modpack and our server updated to 1.10.2 beta4.1 and I reinstalled and every time it fails to launch the pack. I tried reinstalling the instance and then the At Launcher itself and still the same problem. Crash report is here: http://pastebin.com/KLx68HJG

**Comment by Cuppycake** [comment]
Perfect place for it.  Sorry you're having this issue.  

Are you using optifine or were using and removed optifine?  Have you been able to get on the pack on 1.10 versions prior to this or is this the first time you've tried loading it?

**Comment by preflora** [comment]
Whenever I was downloading it I did check the optifine box. I don't believe i removed it at any point. I did today try deleting the instance and re-downloading without adding optifine and it is now launching. After I got it to work I tried re-adding optifine myself to the folder and the same problem as before happened. 
This is the first time I have tried loading the 1.10 of this pack. 

Looks like I can manage to run it for now without optifine thank you for the help.
